simple_rl/agents/func_approx/dsc/OptionClass.py

```diff
@@ -234,7 +235,7 @@ def add_initiation_experience(self, states):
 		if len(states) >= self.buffer_length:
 			segmented_states = segmented_states[-self.buffer_length:]
 		if self.pixel_observations:
-			examples = [np.array(segmented_state.features())[-1, :, :] for segmented_state in segmented_states]
+			examples = [(np.array(segmented_state.features())[-1, :, :]) for segmented_state in segmented_states]
```

Comment:
Can we add a convenience function inside the OptionClass that takes a state object and returns a numpy array which represents the input the initiation classifier.

Reply:
For example:

`def f(x, n):
    return x[-n, ...]`

Reply:
Also, maybe add a comment on why the tuple-ing was needed?